### PR TITLE
Updated user docs with images

### PR DESCRIPTION
**Pull request by alejandrox1**
Added images detailing how to increase the ammount of mem dedicated to
the Docker engine in Mac.

/kind documentation

Signed-off-by: Jorge Alarcon Ochoa <alarcj137@gmail.com>

### CODE REVIEW

File: docs/user/README.md [hunk 1]
```diff
@@ -23,9 +23,13 @@ that you have at least 4GB of RAM and disk space (these are estimates for a
 single node `kind` cluster) dedicated to the virtual machine (VM) running the
 Docker engine otherwise the Kubernetes cluster might fail to start up.
 
-To change the resource limits for the Docker engine you'll need to open the
-**Preferences** menu, go to the **Advanced** settings page, and change the
+To change the resource limits for the Docker engine, in Mac, you'll need to open the
```
Comment (neolit123) [inline docs/user/README.md]: i would amend this to:
> To change the resource limits for the Docker engine on Mac, you'll need to open the...

**Comment by BenTheElder** [comment]
@alejandrox1 you should consider joining the org, I'd be happy to sponsor :-)
https://github.com/kubernetes/community/blob/master/community-membership.md

**Pull request merged**